src/scaffold/app/nix.rs

```diff
@@ -42,16 +42,9 @@ pub fn flake_nix(holo_enabled: bool, package_manager: &PackageManager) -> FileTr
       formatter = pkgs.nixpkgs-fmt;
 
       devShells.default = pkgs.mkShell {{
-        inputsFrom = [ inputs'.holonix.devShells ];
-
-        packages = (with inputs'.holonix.packages; [
-          holochain
-          lair-keystore
-          hc-launch
-          hc-scaffold
-          hn-introspect
-          rust # For Rust development, with the WASM target included for zome builds
-        ]) ++ (with pkgs; [
+        inputsFrom = [ inputs'.holonix.devShells.default ];
```

Comment:
This makes sense. We don't have to expose all the flexibility of Holonix to scaffolding users right away. If they want to make changes then they can go exploring and use documentation from the Holonix repo